src/modules/dashboard/components/Dialogs/CancelStreamingDialog/CancelStreamingForm.tsx

```diff
@@ -0,0 +1,287 @@
+import React from 'react';
+import classNames from 'classnames';
+import { defineMessages, FormattedMessage } from 'react-intl';
+import { FormikProps } from 'formik';
+import { ROOT_DOMAIN_ID } from '@colony/colony-js';
+
+import { Colony, useLoggedInUser } from '~data/index';
+import Dialog, { DialogSection } from '~core/Dialog';
+import { Annotations, InputLabel, Toggle } from '~core/Fields';
+import Heading from '~core/Heading';
+import Button from '~core/Button';
+import UserAvatar from '~core/UserAvatar';
+import UserMention from '~core/UserMention';
+import { useTransformer } from '~utils/hooks';
+import { getAllUserRoles } from '~modules/transformers';
+import { hasRoot } from '~modules/users/checks';
+import { useDialogActionPermissions } from '~utils/hooks/useDialogActionPermissions';
+import MotionDomainSelect from '~dashboard/MotionDomainSelect';
+
+import { FormValues } from './CancelStreamingDialog';
+import FundingSourceItem from './FundingSourceItem';
+import styles from './CancelStreamingDialog.css';
+
+const MSG = defineMessages({
+  header: {
+    id: 'dashboard.CancelStreamingDialog.CancelStreamingForm.header',
+    defaultMessage: 'Cancel streaming payment',
```

Comment:
First letter of each word in 'Cancel Streaming Payment' should be capitalized.

Reply:
it is fixed